cc/tf_utils.cc

```diff
@@ -97,11 +96,7 @@ void WriteTfExamples(const std::string& path,
 
 }  // namespace
 
-void WriteGameExamples(const std::string& output_dir,
-                       const std::string& output_name,
-                       const MctsPlayer& player) {
-  MG_CHECK(file::RecursivelyCreateDir(output_dir));
-
+std::vector<tensorflow::Example> FetchExamples(const MctsPlayer& player) {
```

Comment:
`FetchExamples` sounds like it's reading them from somewhere: the first bit of this pull request I started reviewing was tf_bt_utils.cc and I initially thought that `FetchExamples` was reading from BigTable.

How about something like `MakeExamples` or `CreateExamples`?